instrumentation/vertx/vertx-kafka-client-3.5/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("io.vertx")
+    module.set("vertx-kafka-client")
+    versions.set("[3.5.0,)")
+    assertInverse.set(true)
+    extraDependency("org.apache.kafka:kafka-clients:0.11.0.0")
+  }
+}
+
+dependencies {
+  bootstrap(project(":instrumentation:kafka:kafka-clients:kafka-clients-0.11:bootstrap"))
+  implementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-common:library"))
+
+  library("io.vertx:vertx-kafka-client:3.5.0")
+
+  // vertx-codegen and vertx-docgen dependencies are needed for Xlint's annotation checking
+  library("io.vertx:vertx-codegen:3.0.0")
+  testLibrary("io.vertx:vertx-docgen:3.0.0")
+
+  // vertx-kafka-client 3.5 uses kafka-clients 0.10.2.1 by default, need to bump it to make instrumentation work
+  testImplementation("org.apache.kafka:kafka-clients:0.11.0.0")
```

Comment:
I'd support naming this `vertx-kafka-client-3.6` for a simpler documentation/support story, and updating the base test version to 3.6

Reply:
Done 👍